.storybook/preview.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
+
+const cssVars = `
```

Comment:
Discussed with Stuart how this is duplication of code in `athens.style`, and how we could put it in a JS file and use it as the source of truth for both setups. Left it as is for now to reduce scope of the PR.

Reply:
Not sure about the whole of athens.style, but at least part of it yes.

Reply:
Yeah just the part that is all the CSS properties.

Reply:
This has been implemented in my current branch https://github.com/shanberg/athens/commit/dca06885cc21c086010eacd45c9d3fb8bb492310